### PR TITLE
Use psammead-paragraph

### LINKED ISSUE #997: Pull Paragraph component from Psammead
**Is your feature request related to a problem? Please describe.**
Once the component is added into Psammead...:

https://github.com/BBC-News/psammead/issues/52

...we should remove the component from Simorgh and import the Psammead version instead.

**Describe the solution you'd like**
A single PR which removes our code for this component and pulls in the external one.
 
**Describe alternatives you've considered**
N/A

**Testing notes**
[Tester to complete]

Dev insight: Pages should look the same - regression check please.

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed)

### PR DESCRIPTION
**Pull request by sareh**
Resolves #997

This PR uses `@bbc/psammead-paragraph`
Updated snapshots.

**Testing instructions**
```
git checkout use-psammead-paragraph
npm ci
npm run dev
```
Visit http://localhost:7080/news/articles/c9rpqy7pmypo

Compare with https://www.test.bbc.com/news/articles/c9rpqy7pmypo

The paragraphs should be the same.

- [ ] ~Tests added for new features~
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

**Comment by sareh** [comment]
Failures in the build are due to https://www.npmjs.com/package/@bbc/psammead-paragraph not yet being public.

**Review by bcmn — DISMISSED** [review]

**Review by dr3 — DISMISSED** [review]

**Review by pjlee11 — DISMISSED** [review]

**Review by bcmn — DISMISSED** [review]

File: src/app/components/Text/__snapshots__/index.test.jsx.snap [hunk 1]
```diff
@@ -1,24 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```
Comment (dr3) [inline src/app/components/Text/__snapshots__/index.test.jsx.snap]: Snapshot needs updating

File: src/app/components/Text/__snapshots__/index.test.jsx.snap [hunk 2]
```diff
@@ -6,15 +6,6 @@ exports[`Text should render bold 1`] = `
   font-family: ReithSansNewsRegular,Helvetica,Arial,sans-serif;
   padding-bottom: 1rem;
   margin: 0;
-  font-size: 0.9375em;
```
Comment (bcmn) [inline src/app/components/Text/__snapshots__/index.test.jsx.snap]: These shouldn't have been removed from the snapshot, following [psammead#164](https://github.com/BBC-News/psammead/pull/164). I think it's a case of the cached version of the Psammead package being used -- bumping the requested version in `package.json` immediately causes these snapshots to fail.

File: package.json [hunk 3]
```diff
@@ -58,6 +58,7 @@
   },
   "homepage": "https://github.com/bbc-news/simorgh#readme",
   "dependencies": {
+    "@bbc/psammead-paragraph": "^0.1.1",
```
Comment (bcmn) [inline package.json]:
```suggestion
    "@bbc/psammead-paragraph": "^0.1.2",
```

See above.

**Review by bcmn — CHANGES_REQUESTED** [review]
Otherwise looks good! 👍

**Review by bcmn — DISMISSED** [review]

**Review by dr3 — DISMISSED** [review]
👍

**Review by dr3 — APPROVED** [review]

**Review by ChrisBAshton — APPROVED** [review]

**Pull request merged**